net.jenyay.Outwiker.yml

```diff
@@ -0,0 +1,36 @@
+app-id: net.jenyay.Outwiker
+runtime: org.gnome.Platform
+runtime-version: '3.36'
+sdk: org.gnome.Sdk
+command: outwiker
+rename-icon: net.jenyay.Outwiker
+finish-args:
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=ipc
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=host
```

Comment:
do you really need host access?

Reply:
I need access to home directory, mouned disks and flash drives.